Library/ViewModels/PledgeViewUseCase.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+import KsApi
+import ReactiveSwift
+
+public protocol PledgeViewUseCaseInputs {
+  func goToPledgeViewTapped()
+}
+
+public protocol PledgeViewUseCaseOutputs {
+  var goToNativePledgeView: Signal<ManagePledgeViewParamConfigData, Never> { get }
+  var goToPledgeManagementViewPledge: Signal<URL, Never> { get }
+}
+
+public protocol PledgeViewUseCaseType {
+  var inputs: PledgeViewUseCaseInputs { get }
+  var outputs: PledgeViewUseCaseOutputs { get }
+}
+
+public final class PledgeViewUseCase: PledgeViewUseCaseType, PledgeViewUseCaseInputs,
+  PledgeViewUseCaseOutputs {
+  public init(with projectAndBacking: Signal<(Project, Backing), Never>) {
+    let goToPledgeViewSignal = self.goToPledgeViewTappedProperty.signal
+
+    let shouldGoToPMPledgeView = projectAndBacking
```

Comment:
nit: `shouldGoToPledgeManagementPledgeView` just to keep the same convention as the other names in this flow